private/model/api/codegentest/service/awsquerycompatible/awsquerycompatible_test.go

```diff
@@ -0,0 +1,20 @@
+//go:build go1.8 && codegen
+// +build go1.8,codegen
+
+package awsquerycompatible
+
+import ( "testing" )
+
+func Test_MappedErrorCode(t *testing.T) {
+	assertErrorCode("AWS.SimpleQueueService.QueueDeletedRecently", ErrCodeQueueDeletedRecently, t)
```

Comment:
Generally we don't use helpers like this for simple compares, because it makes the line the test failed on little bit more difficult to track down. you could use the [t.Helper()]() function to tell the go test to ignore this function call when reporting the error. Instead go test will report the error on the next function up the stack, e.g. line 9 instead of line 17.

Alternative is to use an inline pattern which we commonly use.

```go
if e, a := "AWS.SimpleQueueService.QueueDeletedRecently", ErrCodeQueueDeletedRecently; e != a {
    t.Errorf("expect %v code, got %v", e, a)
}
```

That's slightly more verbose, but make it easier to update, change, or track down the test in the future.

Reply:
Awesome, I will update the PR.